docs/templates/parameters.md

```diff
@@ -248,12 +248,14 @@ data "coder_parameter" "cpu" {
 
 As a template improvement, the template author can consider making some of the new `coder_parameter` resources `mutable`.
 
-## Managed Terraform variables
+## Terraform template-wide variables
+
+> ⚠️ Flag `feature_use_managed_variables` is available until v0.25.0 (Jul 2023) release. After this release, template-wide Terraform variables will be enabled by default.
```

Comment:
Should this mention the coder provider version as well?

Reply:
In theory, terraform-provider-coder still accepts the `feature_use_managed_variables`, so maybe not overcomplicate it :)